easybuild/easyconfigs/n/NewHybrids/NewHybrids-1.1_Beta3-GCC-10.2.0.eb

```diff
@@ -0,0 +1,42 @@
+# Author: Denis Krišťák (INUITS)
+# License: GPLv2
+
+easyblock = 'Binary'
+
+name = 'NewHybrids'
+version = '1.1_Beta3'
+
+homepage = 'https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs'
+description = """This implements a Gibbs sampler to estimate the posterior probability that genetically sampled
+individuals fall into each of a set of user-defined hybrid categories."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/eriqande',
+        'repo_name': 'newhybrids',
+        'commit': '6fc8fd9ca6e64ee0be08af12ec8bfe73a7de00cc',
+        'recursive': True,
+    }
+}]
+checksums = [None]
+
+extract_sources = True
+
+install_cmd = '%(builddir)s/newhybrids/Compile-with-no-gui-Linux.sh && '
```

Comment:
Patch the install script (via a `sed` command) to ensure the right compiler command and options are used?